catalog/tests/dags/providers/provider_api_scripts/test_science_museum.py

```diff
@@ -107,6 +107,56 @@ def test_get_query_param_offset_page_number():
     assert actual_param == expected_param
 
 
+@pytest.mark.parametrize(
+    "record",
+    [
+        # missing foreign_landing_url
+        {"links": {}},
+        {"links": {"self": ""}},
+        {"links": {"self": "link"}, "attributes": {"multimedia": []}},
+    ],
+)
+def test_get_record_data_returns_none_for_falsy_fid_and_landing_url(record):
```

Comment:
```suggestion
def test_get_record_data_returns_none_for_falsy_landing_url_and_multimedia(record):
```

Falsy fid is not tested in this case, right?